src/util/defined.js

```diff
@@ -0,0 +1,11 @@
+// Checks that passes properties are 'defined', meaning that calling them with d, i returns non null values
+export default function(d, i) {
+    for (var c = 0, j = arguments.length; c < j; c++) {
+        if (arguments[c] !== d && arguments[c] !== i) {
```

Comment:
It makes perfect sense, I will refactor, move it to fc.util.fn and roll-out the usage.